mobile/src/main/java/org/openhab/habdroid/util/MyHttpClient.java

```diff
@@ -118,6 +120,17 @@ public void setBasicAuth(final String username, final String password, boolean p
         headers.put("Authorization", credential);
     }
 
+    public void setBaseUrl(String baseUrl) {
```

Comment:
Should make this a constructor parameter.

Reply:
Done